scripts/thorium_best_opds_feed/.readme

```diff
@@ -0,0 +1,4 @@
+
+npm install --global surge
+
+surge THIS_FOLDER thorium-reader.surge.sh
```

Comment:
Also ensure `--add EMAIL_ADDRESS` to ensure multiple "admins" can remotely control the contents of the deployed list of OPDS feeds (otherwise, only a single developer can edit this data, which can be problematic if an urgent change is required, and the developer is not reachable).

https://surge.sh/help/adding-collaborators

Reply:
What is your email address ?

Reply:
`daniel.weck@gmail.com`